### DISCUSSION TITLE
What is a prototype chain in JavaScript?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#339] [Q&A] [answered]
In short.

**Comment by d4tdev** [comment, marked as answer]
The prototype object has prototype of its own and so on until an object is reached with null as its prototype. So, this linking with one another is called as prototype chain.Null has no prototype and it acts as a final link in this prototype chain.

**Comment by Preetiraj3697** [comment]
The prototype of an object would also have a prototype object. This continues until we reach the top level when there is no prototype object. This is called prototype chaining or prototype chain in JavaScript. The properties defined on the prototype objects are also accessible to the object instance.
